### PR TITLE
Fee impl in progress

### PR DESCRIPTION
**Pull request by TayGov**
# Description

This PR includes the following proposed change(s):

Fee service implementation

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?
postman and unit tests
## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java [hunk 1]
```diff
@@ -166,4 +169,13 @@ public EfilingError buildEfilingError(ErrorResponse errorResponse) {
         return response;
     }
 
+    private BigDecimal getFee(String type) {
+        try {
+            return efilingLookupService.getServiceFee(type).getFeeAmt();
+        } catch (DatatypeConfigurationException  e) {
```
Comment (akroon3r) [inline src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java:175]: whats up with the red bar <- from 175 to 177

Comment (TayGov) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java:175]: Ahh we aren't testing the exception

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImplTest.java [hunk 2]
```diff
@@ -111,14 +117,17 @@ public void setUp() {
         when(submissionServiceMock.getByKey(Mockito.eq(CASE_8)))
                 .thenReturn(Optional.of(submissionNoCsoAccount));
 
-        sut = new SubmissionApiDelegateImpl(submissionServiceMock, navigationProperties, cachePropertiesMock, submissionMapperMock, feeServiceMock, efilingAccountServiceMock);
+        ServiceFee serviceFee = new ServiceFee();
+        serviceFee.setFeeAmt(BigDecimal.valueOf(2));
```
Comment (akroon3r) [inline src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImplTest.java:121]: does 2 have a meaning or is it just a value

File: src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java (hunk 1 above)
Comment (akroon3r) [reply src/backend/efiling-api/src/main/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImpl.java:175]: FOUND THE 1%

File: src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImplTest.java (hunk 2 above)
Comment (TayGov) [reply src/backend/efiling-api/src/test/java/ca/bc/gov/open/jag/efilingapi/submission/SubmissionApiDelegateImplTest.java:121]: Just the value of two. When using bigdecimal we can't just assign it an int

File: src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/CSOLookupServiceImpl.java [hunk 3]
```diff
@@ -18,36 +16,29 @@
...
-            getServiceFeeElement.setString1(serviceId);
-            getServiceFeeElement.setDate2(Date2XMLGregorian(new Date()));
-
-            GetServiceFeeResponseElement response = lookupFacadeItf.getServiceFee(getServiceFeeElement);
-            serviceFees = SetServiceFees(response.getResult());
-
+            response = lookupFacadeItf.getServiceFee(serviceId,Date2XMLGregorian(new Date()));
         }
-        catch(DatatypeConfigurationException e) {
+        catch(DatatypeConfigurationException | NestedEjbException_Exception e) {
```
Comment (akroon3r) [inline src/backend/libs/efiling-lookup-client/src/main/java/ca/bc/gov/open/jag/efilinglookupclient/CSOLookupServiceImpl.java:36]: Same as above

**Pull request merged**